shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -65,7 +65,9 @@ class FlutterPlatformViewsController {
   // Discards all platform views instances and auxiliary resources.
   void Reset();
 
-  bool SubmitFrame(bool gl_rendering,
+    void extracted();
```

Comment:
what's that?

Reply:
sorry, this is prob caused by xcode auto extraction tool. I am surprised it didn't come up when compile.

Reply:
oh it is in header.